deploy/charts/openebs-monitoring/dashboards/cStor/cstor-volumes.json

```diff
@@ -1034,11 +1069,38 @@
         "skipUrlSync": false,
         "type": "datasource"
       },
+      {
+        "allValue": null,
+        "current": {},
+        "datasource": null,
+        "definition": "label_values(openebs_size_of_volume,openebs_cstor_label)",
+        "description": null,
+        "error": null,
+        "hide": 2,
+        "includeAll": false,
+        "label": "openebs_cstor_label",
+        "multi": false,
+        "name": "openebs_cstor_label",
+        "options": [],
+        "query": {
+          "query": "label_values(openebs_size_of_volume,openebs_cstor_label)",
```

Comment:
Is this working fine? Having query inside query? Can you please once import these whole dashboard and confirm. I am a bit doubtful whether this would work fine when imported.

Reply:
Yes, it's working fine. I have already check.